docs/reference/runtimes/python/python-reference.md

```diff
@@ -54,20 +58,30 @@ ADD ./my-function.yaml /etc/nuclio/config/processor/processor.yaml
 CMD [ "processor" ]
 ```
 
-### Notes
+<a id="function-configuration"></a>
+## Function configuration
+
+Your function-configuration file (for example, `my-function.yaml` for the [example Dockerfile](#dockerfile)) must include the name of your handler function and Python runtime. For more information, see the [function-configuration reference](/docs/reference/function-configuration/function-configuration-reference.md). For example:
 
-- Make sure to change both `my-function-code` & `my-function.yaml` to your real function code & configuration
-- `my-function.yaml` should at least include your handler & runtime, in example:
 ```yaml
 spec:
   handler: main:handler
   runtime: python:3.6
+```
+
+<a id="build-and-execution"></a>
+## Build and execution
+
+Following are example commands for building and running the latest version of a `my-function` function that's listening on port 8090;
+replace the function name, version and the port number, as needed:
+
+Building and Running example commands:
 
+```sh
+docker build -t my-function:latest .
+docker run --name my-function --rm -d -p 8090:8080 my-function:latest
 ```
 
-Building & Running example commands:
-`docker build -t my-function:latest .`
-`docker run --name my-function --rm -d -p 8090:8080 my-function:latest`
+That's it, your Nuclio function should be running and listening on port 8090
```

Comment:
I intentionally removed this line in my previous review comment; it's not necessary now that the new section starts with "Following are example commands for building and running the latest version of a `my-function` function that's listening on port 8090". If your adamant about this, we would need to edit the sentence (grammar wise) and also edit the opening sentence in this section to avoid redundancy.